app/controllers/approvals_controller.rb

```diff
@@ -6,7 +6,7 @@ def create
 
     current_student.add(@approvable)
 
-    @graduated = current_student.total_credits >= 450 && previous_total_credits < 450
+    @graduated = current_student.total_credits >= 450 && previous_total_credits < 450 && groups_credits_met?
```

Comment:
and what i meant is that we are doing `previous_total_credits = current_student.total_credits` to check that the user was not graduated before doing the `current_student.add(@approvable)`, so we are checking the user changed from < 450 to >= 450 credits, but we are onlyn checking `groups_credits_met? ` after adding the approvable.

So my concern is that if the user was already over 450 but didn't meet the min credits per group, and then adds a new approvable to meet the min per group, `@graduated` will be false

Reply:
Oh you are right. I'll change that. Thanks

Reply:
Updated in 6c77365733c372e7f64a1d4d651f8af9e9164153